Results global comparison (#10)
greenelab/annorxiver_manuscript: content/04.results.md

### Comparing bioRxiv to PubMed Central

#### Global Comparison of bioRxiv and PubMed Central 

![

> BioRxiv has more category specific terms whereas PubMed Central has more patient oriented articles.

Reviewer: Is this saying that there are different topic areas, or something else that I'm missing?

The plot on the left (A) is a point range plot of the odds ratio with respect to bioRxiv.
Values greater than one indicate a high association with bioRxiv whereas values less than one indicate high association with PubMed Central.
The dotted line provides a breaking point between both categories.
The plot on the right (B) is a barchart of token frequency appearing in bioRxiv and PMC respectively.
](https://raw.githubusercontent.com/greenelab/annorxiver/e32661b855cc6622cc1138fcd1606ef3500f0ff9/biorxiv/corpora_comparison/output/figures/biorxiv_vs_pubmed_central.png){#fig:biorxiv_pmc_global_comparison}

> The linguistic style of preprints can be compellingly different than published peer reviewed articles.

Reviewer: ```suggestion
The linguistic style of the bioRxiv corpus differs from the PMC corpus.
```

> We quantified the difference between preprints in biorxiv to published articles in PubMed Central (PMC) by calculating the odds ratio on tokens appearing in both repositoies respectively.

Reviewer: ```suggestion
We compared preprints in bioRxiv and published manuscripts in PMC.
We found that tokens such as (PUT TOKENS HERE) had a high odds ratio, indicating enrichment in (PMC or bioRXiv?) (Figure {@fig:biorxiv_pmc_global_comparison}).
```

> Tokens with a high odds ratio are topic-specific words such as: "neuron", "genome", "RNA" and "network" (Figure {@fig:biorxiv_pmc_global_comparison}).

Reviewer: Can you try this starting framing and then follow-on with the low odds ratio and close with an interpretation of your findings? Do you think this is how people are writing, typesetting, or the topic areas included/absent from bioRxiv?

### Comparing bioRxiv to PubMed Central

#### Global Comparison of bioRxiv and PubMed Central 

![
BioRxiv is more focused on biological discoveries rather than disease treatments and clinical trials.

> The plot on the left (A) is a point range plot of the odds ratio with respect to bioRxiv.

Reviewer: Are there confidence intervals on this? I can't tell if they are there but so small because of the total sample size that they don't appear, or if they are not present.

Author: There are confidence intervals; however, they are quite tiny (order of second or third decimal place).

The dotted line provides a breaking point between both categories.
The plot on the right (B) is a bar chart of token frequency appearing in bioRxiv and PMC respectively.
](https://raw.githubusercontent.com/danich1/annorxiver/f5d6c2d04e2fef0d38eaee77ef6c2a0b19ef6358/biorxiv/corpora_comparison/output/figures/biorxiv_vs_pubmed_central.png){#fig:biorxiv_pmc_global_comparison}

The linguistic style of the bioRxiv corpus differs from the PMC corpus.
We compared preprints in bioRxiv and published manuscripts in PMC.
We found that tokens such as "neuron", "genome", "RNA" and "network" had a high odds ratio, indicating enrichment in bioRxiv (Figure {@fig:biorxiv_pmc_global_comparison}).
Likewise, we found tokens such as "patient", "health", $\plusminus$, and "ml" to have a low odds ratio, indicating enrichment in PMC (Figure {@fig:biorxiv_pmc_global_comparison}).
This separation of tokens suggests a prevalence of articles focused on clinical trials and patient research within PMC compared to bioRxiv.
Furthermore, bioRxiv has a predominance of neuroscience and bioinformatic topics.

> In regard to writing, citation styles diversify from the familiar "et al." form as preprints transition through the publication process.

Reviewer: Do you want to add one sentence that notes there also appear to be typesetting differences? The +/- sign appears to be one of these.
